### ISSUE TITLE
Using align="START" for FormArrays will not be rendered

### ISSUE THREAD

**Issue by Karamuto** [udos86/ng-dynamic-forms#663]
## I'm submitting a

<pre><code>
[x] Bug / Regression
[ ] Feature Request / Proposal
[ ] Question
</code></pre>

## I'm using

<pre><code>
NG Dynamic Forms Version: `5.4.3`

[x] Core
[ ] Basic UI
[ ] Bootstrap UI  
[ ] Foundation UI
[ ] Ionic UI
[ ] Kendo UI
[x] Material  
[ ] NG Bootstrap
[ ] Prime NG
</code></pre>

## Description

As the title says, using `align="START"` does not work as intended for FormArrays, in my local example:

```
<form [formGroup]="formGroup">
  <dynamic-material-form [group]="formGroup"  [model]="formModel">
     <ng-template modelId="formArray" align="START" let-id="id">

      <span>This shows before every group in '{{id}}'</span>
   </ng-template>
   <ng-template modelId="formArray" let-id="id">
      <span>This shows after every group in '{{id}}'</span>
    </ng-template>
</dynamic-material-form>
</form>
```

does show the second template but not the first.

**Comment by udos86** [comment]
@Karamuto Hi!

This is currently a known restriction to form array templates.

@everybody Let me know if this is an desirable feature

**Comment by udos86** [comment]
closed as duplicate of #609
